diag_manager/diag_manager.F90

```diff
@@ -1647,6 +1647,7 @@ LOGICAL FUNCTION send_data_3d(diag_field_id, field, time, is_in, js_in, ks_in, &
        IF ( fms_error_handler('diag_manager_mod::send_data_3d', err_msg_local, err_msg) ) RETURN
     END IF
     if (use_modern_diag) then !> Set up array lengths for remapping
+      field_modern => null()
```

Comment:
You should use `nullify` here I think...

Reply:
i think they do the same thing